### DISCUSSION TITLE
Better for the backend?

### DISCUSSION THREAD

**Discussion by DanielAlmeidaToledo** [community/community#54304] [Programming Help] [answered]
### Body

Which is better for the backend, c# or node.js?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rafahcerqueira** [comment, marked as answer]
This depends on several factors, such as what is the purpose of your application, what features you will need, among others. But I particularly like NodeJS 😄
